src/main/kotlin/no/nav/syfo/application/mq/MQSender.kt

```diff
@@ -0,0 +1,57 @@
+package no.nav.syfo.application.mq
+
+import com.ibm.msg.client.wmq.common.CommonConstants.*
+import no.nav.syfo.Environment
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import javax.jms.JMSContext
+import javax.jms.Message
+
+private val log: Logger = LoggerFactory.getLogger("no.nav.syfo.application.mq")
+
+interface MQSenderInterface {
+    fun sendArena(payload: String)
+    fun sendReceipt(payload: String)
+    fun sendBackout(message: Message)
+}
+
+class MQSender(
+    val env: Environment,
+) : MQSenderInterface {
```

Comment:
Brukes `MQSenderInterface` flere steder? Hvis ikke, trenger vi eget interface da? 🤷‍♂️

Reply:
Ja, det er enklere å lage tester når man kan mocke interface't. Se BlockingApplicationRunnerSpek.

Reply:
Hmm, det er sikkert noe jeg overser, men vil det ikke være akkurat det samme å gjøre `mockk<MQSender>` som `mockk<MQSenderInterface>`? Ikke at det er krise med interface uansett, men fint om jeg også skjønner hvilke fordeler det gir 👍

Reply:
Ja, tror kanskje det blir litt enklere siden da trenger man ikke å bale med å få den interne koden i MQSender (feks jmsContext) til å funke for testene, men det er mulig at du har rett i at det ville blitt det samme i dette tilfellet.